Dockerfile.aarch64

```diff
@@ -23,7 +23,7 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:trusty
+FROM aarch64/ubuntu:vivid
```

Comment:
Yes with wily it works. See changes below. Should I send patch with wily?

-FROM aarch64/ubuntu:trusty
+FROM aarch64/ubuntu:wily
 
 # Packaged dependencies
 RUN apt-get update && apt-get install -y \
@@ -45,15 +45,15 @@ RUN apt-get update && apt-get install -y \
        libc6-dev \
        libcap-dev \
        libsqlite3-dev \
-       libsystemd-journal-dev \
+       libsystemd-dev \
        mercurial \